specfile/changelog.py

```diff
@@ -287,22 +305,20 @@ def parse_evr(s):
         if since is None:
             start_index = 0
         else:
+            since_evr: EVR = parse_evr(since) if isinstance(since, str) else since
```

Comment:
I think it would be nicer to extend `parse_evr()` to accept `EVR` or string and avoid this.